modules/ocf_stats/manifests/init.pp

```diff
@@ -3,9 +3,7 @@
 
   class { '::apache':
     default_vhost => false,
-    mpm_module    => 'prefork'
```

Comment:
Nice, this is good to remove, the event mpm should work fine with this I imagine